### DISCUSSION TITLE
Persistent toggles between neovim sessions

### DISCUSSION THREAD

**Discussion by ChrisGVE** [LazyVim/LazyVim#5007] [Extra Requests]
I am not sure if this topic has been raised before (and I've searched but found nothing):

Would there be a way to make toggles persistent, e.g., in the UI menu? In general, would it be possible to keep track of the state of each toggle? This could be done globally or even per project.

Maybe that's possible, but again, I searched and could not find out if that's the case.

What do you guys think about that?

**Comment by ghost** [comment]
https://github.com/folke/snacks.nvim/discussions/440
